frontend/src/main/components/Commons/FarmStats.js

```diff
@@ -15,13 +15,13 @@ const FarmStats = ({userCommons}) => {
                 <img class="icon" src={Cash} alt="Cash"></img>
```

Comment:
Before I can even review this on its merits, a couple of things need to happen:

* If this is truly only about backend changes, then the only changes should be to backend files.
* There should not be a .DS_Store file in a commit to a repo.  Please remove it (and feel free to add .DS_Store to .gitignore if that helps you avoid this mistake in the future.
* If changes to frontend code ARE necessary, then you need to explain those changes AND make sure they pass CI/CD testing

Reply:
Removed frontend changes and removed the .DS_Store. Additionally, I updated the `.gitignore` so .DS_Store files should be automatically ignored from now on

Reply:
The changes are only on backend now and the tests are passed